### PR TITLE
#73/team standup card with styling

### LINKED ISSUE #73: Team Standup View Component
child of #79
child of #78

renders the team standup details as a card

### PR DESCRIPTION
**Pull request by serpient**
### Live Test Render
image

### Usage
```
<TeamStandupComponent userStandup={standupData}/>
```

Example Data passing in:
```
let standupData = {
  "id": "MXN0YW5kdXA=",
  "type": "NewsfeedStandup",
  "timestamp": 1535088249192,
  "user": {
    "id": "6",
    "username": "the-vampiire",
    "avatar": "https://avatars3.githubusercontent.com/u/25523682?v=4"
  },
  "standup": {
    "id": "1",
    "is_expired": false,
    "progress_sentiment": "green",
    "worked_on": `Worked on creating forms for project-update. trying to style mockups for newsfeed`,
    "working_on": `Will work on coding the newsfeed and finish the project portal. need to finish new forms`,
    "blocked_on": `Figuring out how to hook up these forms without dynamic forms. `,
    "expiration": 1535692113990
  }
}
```

### CODE REVIEW

**Comment by luoto** [comment]
What is the difference between this and #73?

**Comment by serpient** [comment]
need to refactor component so it expects this object:
```
{
        "id": "MXN0YW5kdXA=",
        "type": "NewsfeedStandup",
        "timestamp": 1535088249192,
        "user": {
          "id": "6",
          "username": "the-vampiire",
          "avatar": "https://avatars3.githubusercontent.com/u/25523682?v=4"
        },
        "standup": {
          "id": "1",
          "is_expired": false,
          "progress_sentiment": "green",
          "expiration": 1535692113990
        }
      },
```

File: src/components/FeedPortal/components/TeamStandupCard.jsx [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+  }
+  let standupObjects = [
+    { label: 'Health Status:', data: healthStatus },
+    { label: 'Worked on:', data: standup.worked_on },
+    { label: 'Working on next:', data: standup.working_on },
+    { label: 'Blockers:', data: standup.blocked_on }
+  ]
+
+  let renderedStandup = standupObjects.map((standup, idx) => {
+    let className = "team-standup-answer"
```
Comment (luoto) [inline src/components/FeedPortal/components/TeamStandupCard.jsx]: Suggestion:
- Clean up with ternary

File: src/components/FeedPortal/components/TeamStandupCard.jsx [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+      healthStatus = 'Nervous';
+      break;
+    case 'green':
+      healthStatus = 'Great!';
+      break;
+    default:
+      healthStatus = 'No Response Yet';
+      break;
+  }
+  let standupObjects = [
```
Comment (luoto) [inline src/components/FeedPortal/components/TeamStandupCard.jsx]: Suggestion:
- Remove colons from the text, and place it on line 35
- Downcase values for label, capitalize with styles

**Pull request merged**